Add pub_hwm and zmq_backlog to the tuning guide (#632)
uyuni-project/uyuni-docs: modules/large-deployments/pages/tuning.adoc

[[pub-hwm]]
=== `pub_hwm`

[cols="1,1"]
|===

> | Description          | The maximum number of outstanding messages sent by `salt-master`. If more than this number of messages need to be sent concurrently, communication with clients will slow down, potentially resulting in timeout errors during load peaks.

Reviewer: ```suggestion
| Description          | The maximum number of outstanding messages sent by `salt-master`. If more than this number of messages need to be sent concurrently, communication with clients slows down, potentially resulting in timeout errors during load peaks.
```

> | Tune when            | xref:client-count[Client count] increases significantly and `Salt request timed out. The master is not responding` errors appear when pinging minions during a load peak.

Reviewer 2: ```suggestion
| Tune when            | xref:client-count[Client count] increases significantly and ``Salt request timed out. The master is not responding.`` errors appear when pinging minions during a load peak.
```
I think single tics don't work with adoc.  If yes, we must adjust it later on, too.
We usually call minions "clients".  I'm not sure whether we want to do the same here.

| Value default        | 1000
| Value recommendation | 10000-100000
| Location             | [path]``/etc/salt/master.d/tuning.conf``
| Example              | `pub_hwm: 10000`
| More information     | https://docs.saltstack.com/en/latest/ref/configuration/master.html#pub-hwm, https://zeromq.org/socket-api/#high-water-mark
|===

[[zmq-backlog]]
=== `zmq_backlog`

[cols="1,1"]
|===

> | Description          | The maximum number of allowed client connections that have started but not concluded the opening process. If more than this number of clients connects in a very short time frame, connections will be dropped and clients will experience a delay re-connecting.

Reviewer: ```suggestion
| Description          | The maximum number of allowed client connections that have started but not concluded the opening process. If more than this number of clients connects in a very short time frame, connections are dropped and clients experience a delay re-connecting.
```
